integration-test/src/test/java/uk/gov/gchq/gaffer/integration/impl/GetElementsIT.java

```diff
@@ -652,8 +652,13 @@ private void shouldGetElements(final Collection<Element> expectedElements,
         final Iterable<? extends Element> resultsElement = graph.execute(opElement, user);
 
         // Then
-        ElementUtil.assertElementEquals(expectedElements, resultsSeed, true);
-        ElementUtil.assertElementEquals(expectedElements, resultsElement, true);
+        if (includeEdges && inOutType == IncludeIncomingOutgoingType.INCOMING) {
```

Comment:
In other tests, I managed to fix the assertions so they correctly test the MatchedVertex behaviour. However, GetElementsIT is quite complex and the MatchedVertex has been tested incorrectly for a while so this might require a new issue or larger refactor of the test.

Reply:
Was there an issue created for this?

Reply:
Issue created: https://github.com/gchq/Gaffer/issues/3138